HarryPotter/Screens/Favourite/FavouriteViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  FavouriteViewController.swift
+//  HarryPotter
+//
+//  Created by Adrianna Mikulska on 28/09/2022.
+//
+
+import UIKit
+
+final class FavouriteViewController: UITableViewController {
+    
+    //MARK: - Properties
+    
+    private let cellId = "cellId"
+    private var favouriteViewModel: FavouriteViewModelProtocol
+    private var characters = [Character]() {
```

Comment:
I think you can delete characters, and instead, use `viewModel.characters`. So that viewModel has most of the things.

Then you will need to refresh tableView somewhere else 🤔